lua/mapvote/client/vgui/mapvote_panel.lua

```diff
@@ -241,6 +241,7 @@ function PANEL:Flash( id )
     end )
     timer.Simple( 0.6, function() panel:SetPaintBackgroundEnabled( false ) end )
     timer.Simple( 0.8, function()
+        if not IsValid( panel ) then return end
```

Comment:
you would need to have this on all the timers. could create a local function to toggle paint background while checking IsValid